dask_ml/model_selection/_split.py

```diff
@@ -3,6 +3,8 @@
 import itertools
 import logging
 import numbers
+from distutils.version import LooseVersion
```

Comment:
We use packaging.version.parse.

Reply:
Also, this should all be centralized in `dask_ml._compat`. If something isn't there that you need (like DASK_213) go ahead and define it.